spec/hydra/works_spec.rb

```diff
@@ -74,18 +50,6 @@
       it 'returns true for a works generic file' do
         expect(works_gfile.generic_file?).to be true
       end
-
-      it 'returns that method is not available for a pcdm collection' do
-        expect(pcdm_coll).not_to respond_to(:generic_file?)
-      end
-
-      it 'returns that method is not available for a pcdm object' do
-        expect(pcdm_obj).not_to respond_to(:generic_file?)
-      end
-
-      it 'returns that method is not available for a pcdm file' do
-        expect(pcdm_file).not_to respond_to(:generic_file?)
-      end
```

Comment:
Because they aren't testing code that is in this gem.  They are asserting the non-existence of a method in a different gem.  This doesn't seem like a useful test.